src/calamares/CalamaresWindow.cpp

```diff
@@ -161,6 +161,31 @@ getWidgetNavigation( Calamares::DebugWindowManager*,
     QBoxLayout* bottomLayout = new QHBoxLayout;
     bottomLayout->addStretch();
 
+    QObject::connect( viewManager,
+                      &Calamares::ViewManager::navigationPanelWidgetChanged,
+                      bottomLayout,
+                      [ bottomLayout ]( QWidget* wgt )
+                      {
+                          if ( !bottomLayout )
+                          {
+                              return;
+                          }
+                          auto prevWgt = bottomLayout->parentWidget()->findChild< QWidget* >( "view-custom-item" );
```

Comment:
prefer `auto*` to emphasize it's a pointer, and longer names: `previousCustomWidget` or so.